llvm/test/CodeGen/EraVM/ra_regress.ll

```diff
@@ -1,8 +1,6 @@
 ; RUN: llc -O0 < %s | FileCheck %s
 
-; ModuleID = 'Test_65'
-source_filename = "Test_65"
```

Comment:
Is there a reason for removing `source_filename`?

Reply:
Yes, it gives nothing in terms of understanding the test. Just distracts the reader.